### DISCUSSION TITLE
Uhhh

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [jeffalo/ocular#100] [Q&A] [answered]
WHY IS THERE A RICKROLL IN A FORUM VIEWERRRR https://github.com/jeffalo/ocular/issues/73

**Comment by jeffalo** [comment, marked as answer]
reasons

**Comment by Quantum-Codes** [comment]
-_-

**Comment by retronbv** [comment]
#100

**Reply by Quantum-Codes** [reply]
> #100

This is the topic

**Comment by pikachub2005** [comment]
> WHY IS THERE A RICKROLL IN A FORUM VIEWERRRR #73

Hey quantum!

**Reply by Quantum-Codes** [reply]
Hi

**Reply by pikachub2005** [reply]
Why don't you want to help with ScratchPlus? ;-;

**Reply by Quantum-Codes** [reply]
Exams,.. school stuff... :(

**Reply by jeffalo** [reply]
this isn't the place for that kind of discussion.
